src/app/assets/stylesheets/functions.scss

```diff
@@ -4,3 +4,7 @@
         max-height: inherit;
     }
 }
+
+#function_tags_list {
+    width: 100%;
+}
```

Comment:
avoid adding css unless it's absolutely necessary, is there something in bulma that you can use instead? also what are you trying to do with this width: 100% property?

Reply:
I tried `is-fullwidth` but it didn't work.

Anyway, without `width: 100%` it looks like this
![image](https://user-images.githubusercontent.com/1307905/83072957-ead71680-a06f-11ea-968f-312b69d1af37.png)

With it
![image](https://user-images.githubusercontent.com/1307905/83073003-fc202300-a06f-11ea-9839-a2a5bc1a725f.png)